canonical/lxd-ui
# Add profile summary panel (side panel) [WD-3020]

## Done

- Added profile summary (side panel)

Fixes [WD-3020](https://warthogs.atlassian.net/browse/WD-3020)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - Open profile list page
    - Click on the description column or the used by column to open the side panel
    - Check that the panel looks and behaves as expected

[WD-3020]: https://warthogs.atlassian.net/browse/WD-3020?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
